Testing/SystemTests/tests/framework/ISIS_PowderPolarisTest.py

```diff
@@ -121,6 +121,92 @@ def cleanup(self):
             mantid.mtd.clear()
 
 
+class FocusTestAbsorptionPaalmanPings(systemtesting.MantidSystemTest):
+
+    focus_results = None
+    existing_config = config['datasearch.directories']
+
+    def requiredFiles(self):
+        return _gen_required_files()
+
+    def runTest(self):
+        # Gen vanadium calibration first
+        setup_mantid_paths()
+        self.focus_results = run_focus_absorption("98533", paalman_pings=True)
+
+    def validate(self):
+        # check output files as expected
+        def generate_error_message(expected_file, output_dir):
+            return "Unable to find {} in {}.\nContents={}".format(expected_file, output_dir,
+                                                                  os.listdir(output_dir))
+
+        def assert_output_file_exists(directory, filename):
+            self.assertTrue(os.path.isfile(os.path.join(directory, filename)),
+                            msg=generate_error_message(filename, directory))
+
+        user_output = os.path.join(output_dir, "17_1", "Test")
+        assert_output_file_exists(user_output, 'POLARIS98533.nxs')
+        assert_output_file_exists(user_output, 'POLARIS98533.gsas')
+        output_dat_dir = os.path.join(user_output, 'dat_files')
+        for bankno in range(1, 6):
+            assert_output_file_exists(output_dat_dir, 'POL98533-b_{}-TOF.dat'.format(bankno))
+            assert_output_file_exists(output_dat_dir, 'POL98533-b_{}-d.dat'.format(bankno))
+
+        assert_output_file_exists(output_dir, "PaalmanPingsCorrections_POLARIS98533_Summed.nxs")
+        for ws in self.focus_results:
+            self.assertEqual(ws.sample().getMaterial().name(), 'Si')
+        self.tolerance_is_rel_err = True
+        self.tolerance = 1e-6
+        return self.focus_results.name(), "ISIS_Powder-POLARIS98533_FocusPaalmanPings.nxs"
+        pass
+
+    def cleanup(self):
+        mantid.mtd.clear()
```

Comment:
I think you also need to delete the PP corrections .nxs file that is output. I just tried and it gets deleted by one of the other tests in this suite but if this test was run on its own the .nxs file would be left behind. I suggest you add the following lines in cleanup here that are present in the other tests:
```
        try:
            _try_delete(output_dir)
            _try_delete(spline_path)
        finally:
            mantid.mtd.clear()
            config['datasearch.directories'] = self.existing_config
```
The final line in here sets the data search config back to its original value and this is worth including so other system tests aren't affected by the code in this unit that sets this to some special POLARIS value